.buildkite/pipeline.build.yml

```diff
@@ -163,6 +163,26 @@
     - if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then python .buildkite/copy_files.py --destination docker_login; fi
     - python ./ci/build/build-docker-images.py --py-versions py310 --device-types cu111 cu112 cu113 cu116 --build-type BUILDKITE --build-base
 
+# NOTE: Python 3.11 is work in progress
```

Comment:
Looks like the PR is ready to be marked as non-draft? Let's make sure the macOS pipeline also works as expected even though we are not adding macos wheel. 

I will get a committer for a review as well. 

nit: Let's delete those(commented codes) in this PR maybe.

Reply:
Lovely - just started it.